### PR TITLE
Refactor LibreNMS directories creation

### PR DESCRIPTION
**Pull request by inetAnt**
- Correct rights for group (add the right to execute)
- Add facl
- Loop on variables instead of copy-pasting code :)

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -200,27 +202,23 @@
...
+    action :create
+    not_if { ::File.exist? dir }
+  end
 
-directory librenms_storagedir do
-  owner librenms_username
-  group librenms_group
-  mode '0755'
-  action :create
+  facl dir do
```
Comment (jmauro) [inline recipes/default.rb]: I don't understand why you have to make sure those acl should hardening and why the directory resource doesn't handle this?

Comment (inetAnt) [reply recipes/default.rb]: I am just following [LibreNMS install guide](https://docs.librenms.org/Installation/Installation-CentOS-7-Apache/), but I guess this is not mandatory for a standard CentOS setup.

Comment (jmauro) [reply recipes/default.rb]: Sorry I still don't understand the acl issue

Comment (inetAnt) [reply recipes/default.rb]: I have removed it

**Review by nicolasgrieco — APPROVED** [review]

**Pull request merged**